challenges/objects-arrays.js

```diff
@@ -46,26 +70,39 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":9,"first_name":"Michail","university":"Universidad Católica de Ávila","email":"mrome8@shinystat.com"},
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
-/* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
+// Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
-Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+// Once you have the new array created, sort the universities alphabetically and log the result.
+
+const universities = graduates.map((student) => student.university).sort();
```

Comment:
We fixed this during our meeting but .sort can be chained on the array that comes from the .map